Source/VMware.vSphereDSC/Tests/Unit/VMHostVss.Unit.Tests.ps1

```diff
@@ -0,0 +1,716 @@
+                            Pnic = $(($script:resourceProperties.Pnic | %{"'$_'"}) -join ',')
+                            PortGroup = $(($script:resourceProperties.Portgroup | %{"'$_'"}) -join ',')
+                        }
+                    )
+                }
+            }
+        )
+'@
+    $script:vssConfigAddVssCode = @'
+        return [VMware.Vim.HostVirtualSwitchConfig] @{
+            Name = '$($script:resourceProperties.VssName)'
+            ChangeOperation = '$($script:netObjectAdd.Operation)'
+            Spec = [VMware.Vim.HostVirtualSwitchSpec] @{
+                Mtu = $($script:resourceProperties.Mtu)
+                NumPorts = $($script:resourceProperties.NumPorts)
+            }
+        }
+'@
+    $script:vssConfigEditVssCode = @'
+        return [VMware.Vim.HostVirtualSwitchConfig] @{
+            Name = '$($script:resourceProperties.VssName)'
+            ChangeOperation = '$($script:netObjectEdit.Operation)'
+            Spec = [VMware.Vim.HostVirtualSwitchSpec] @{
+                Mtu = $($script:resourceProperties.Mtu)
+                NumPorts = $($script:resourceProperties.NumPorts)
+            }
+        }
+'@
+    $script:vssConfigRemoveVssCode = @'
+        return [VMware.Vim.HostVirtualSwitchConfig] @{
+            Name = '$($script:resourceProperties.VssName)'
+            ChangeOperation = '$($script:netObjectRemove.Operation)'
+            Spec = [VMware.Vim.HostVirtualSwitchSpec] @{
+                Mtu = $($script:resourceProperties.Mtu)
+                NumPorts = $($script:resourceProperties.NumPorts)
+            }
+        }
+'@
+    $script:updateNetworkCode = @'
+        return [VMware.Vim.HostNetworkConfigResult] @{
+            ConsoleVnicDevice = @()
+            VnicDevice = @()
+        }
+'@
+
+    $env:PSModulePath = $script:mockModuleLocation
+    $vimAutomationModule = Get-Module -Name VMware.VimAutomation.Core
+    if ($null -ne $vimAutomationModule -and $vimAutomationModule.Path -NotMatch 'TestHelpers') {
+        throw 'The Original VMware.VimAutomation.Core Module is loaded in the current session. If you want to run the unit tests please open a new PowerShell session.'
+    }
+
+    Import-Module -Name VMware.VimAutomation.Core
+
+    $script:viServer = [VMware.Vim.VIServer] @{
+        Name = $script:resourceProperties.Server
+        User = $script:Constants.VIServerUser
+    }
+    $script:networkSystemMoRef = [VMware.Vim.ManagedObjectReference] @{
+        Type = $script:Constants.NetworkSystemMoRefType
+        Value = $script:Constants.NetworkSystemMoRefValue
+    }
+}
+
+function AfterAllTests {
+    Remove-Module -Name VMware.VimAutomation.Core
+    $env:PSModulePath = $script:modulePath
+}
+
+# Calls the function to Import the mocked VMware.VimAutomation.Core module before all tests.
+Try {
+    BeforeAllTests
+
+    Describe 'VMHostVss\Set'  -Tag 'Set' {
+        Context 'Present + VSS does not exist' {
```

Comment:
For better readability change the plus to 'and'.

Reply:
Done